### PR TITLE
Release 0.7.0

### PR DESCRIPTION
**Pull request by mscwilson**
Ready for release.

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: snowplow-tracker.gemspec [hunk 1]
```diff
@@ -9,33 +9,34 @@
...
-  s.license     = 'Apache License 2.0'
-  s.summary     = "Ruby Analytics for Snowplow"
-  s.description = "With this tracker you can collect event data from your Ruby applications, Ruby on Rails web applications and Ruby gems."
-  s.authors     = ["Alexander Dean", "Fred Blundun"]
+  s.license     = 'Apache-2.0'
+  s.summary     = 'Ruby Analytics for Snowplow'
+  s.description = 'With this tracker you can collect event data '\
+                  'from your Ruby applications, Ruby on Rails web'\
+                  ' applications and Ruby gems.'
+  s.authors     = ['Alexander Dean', 'Fred Blundun']
```
Comment (mscwilson) [inline snowplow-tracker.gemspec:29]: Should this be `Snowplow Analytics Ltd` as well?

Comment (paulboocock) [reply snowplow-tracker.gemspec:29]: Yeah, I think so.

**Review by paulboocock — APPROVED** [review]
LGTM! Amazing release, great work.

**Pull request merged**